### DISCUSSION TITLE
Does revolt work in an iFrame?

### DISCUSSION THREAD

**Discussion by LAZERPOINT** [stoatchat/discussions#345] [Q&A]
Hello, for a little while I have been working on an HTML OS that can run websites as apps in an iFrame. So, I was wondering if by any chance I could get revolt to work in the app player. If anyone can answer my question, I would be greatly appreciative of any answers.

**Comment by erkinalp** [comment]
Revolt protocol is open source, so you can use any client. If this question is about Revite, the project's reference client, then ask it in revoltchat/revite.
